columnflow/plotting/plot_util.py

```diff
@@ -1016,3 +1019,58 @@ def remove_label_placeholders(
         sel = "[A-Z0-9]+"
 
     return re.sub(f"__{sel}__", "", label)
+
+
+def calculate_error(
```

Comment:
We generally use this to compute statistical errors, right? If so, we might want to rename this accordingly, e.g. to `calculate_stat_error`. Just "error" sounds rather generic.